modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/FormBuilder/FieldActionsDropDown.es.js

```diff
@@ -48,21 +46,19 @@ class FieldActionsDropDown extends Component {
 	}
 
 	attached() {
-		this._eventHandler.add(dom.on(window, 'resize', this.align.bind(this)));
+		window.addEventListener('resize', this.align.bind(this));
 	}
 
 	close() {
 		this.setState({expanded: false});
 	}
 
 	created() {
-		this._eventHandler = new EventHandler();
-
 		this.close();
 	}
 
 	disposed() {
-		this._eventHandler.removeAllListeners();
+		window.removeEventListener('resize', this.align);
```

Comment:
This call doesn't match the `addEventListener` one, because `this.align !== this.align.bind(this)`.